### PR TITLE
Create description padding based on script name length

### PR DESCRIPTION
**Pull request by SimonBaeumer**

### CODE REVIEW

File: src/Application/Application.php [hunk 1]
```diff
@@ -184,4 +186,17 @@ protected function printHeader(Config $config)
...
+
+    /**
+     * @param Script[] $scripts
+     * @return Int
+     */
+    private function getPaddingSize(array $scripts): Int
+    {
+        $paddingSize = 0;
+        foreach ($scripts as $script) {
+            $paddingSize = strlen($script->getName()) > $paddingSize ? strlen($script->getName()): $paddingSize;
```
Comment (JanPietrzyk) [inline src/Application/Application.php]: please use if statements for better readability

**Review by JanPietrzyk — CHANGES_REQUESTED** [review]

File: src/Application/Application.php [hunk 2]
```diff
@@ -21,6 +21,8 @@ class Application
 
     const RESULT_ERROR = 1;
 
+    const PADDING_SIZE = 30;
```
Comment (JanPietrzyk) [inline src/Application/Application.php]: `const MIN_PADDING_SIZE`

File: src/Application/Application.php [hunk 3]
```diff
@@ -184,4 +188,19 @@ protected function printHeader(Config $config)
...
         }
     }
+
+    /**
+     * @param Script[] $scripts
+     * @return Int
+     */
+    private function getPaddingSize(array $scripts): Int
+    {
+        $paddingSize = 0;
```
Comment (JanPietrzyk) [inline src/Application/Application.php]:
```php
$maxScriptNameLength = 0;
foreach ($scripts as $script) {
   $scriptNameLength = strlen($script->getName());
   if ($scriptNameLength > $paddingSize) {
      $maxScriptNameLength = $scriptNameLength;
   }
}
return $maxScriptNameLength + self::MIN_PADDING_SIZE;
```

a little more care with naming please

**Review by JanPietrzyk — CHANGES_REQUESTED** [review]

File: src/Application/Application.php (hunk 3 above)
Comment (SimonBaeumer) [reply src/Application/Application.php]: done ;)

File: src/Application/Application.php (hunk 2 above)
Comment (SimonBaeumer) [reply src/Application/Application.php]: done ;)

**Review by JanPietrzyk — APPROVED** [review]

**Pull request merged**